Sources/MasKit/AppStore/PurchaseDownloadObserver.swift

```diff
@@ -64,7 +64,7 @@ struct ProgressState {
     let phase: String
 
     var percentage: String {
-        String(format: "%.1f%%", arguments: [floor(percentComplete * 100)])
+        String(format: "%.1f%%", arguments: [percentComplete * 100])
```

Comment:
If I had to guess, I'd say that @argon was trying to ensure `mas` didn't display `0.9995...` as `100.0%`, indicating we're done before we actually are. Maybe this would get us closer to the original intention? (We also no longer need the `arguments:` label or array.)
```suggestion
        String(format: "%.1f%%", floor(percentComplete * 1000) / 10)
```

Reply:
That would make sense, seems like the most reasonnable solution to me!